src/ppx/grfx/grfx_pipeline.cpp

```diff
@@ -85,14 +85,14 @@ grfx::BlendAttachmentState BlendAttachmentState::BlendModeUnder()
 grfx::BlendAttachmentState BlendAttachmentState::BlendModePremultAlpha()
 {
     grfx::BlendAttachmentState state = {};
```

Comment:
oh, sorry, i wasn't clear. you need to add a new blending mode, please do not modify the existing PremultAlpha